c-pallets/file-bank/src/lib.rs

```diff
@@ -91,6 +91,10 @@ pub mod pallet {
 	pub const PACKAGE_FROZEN: &str = "frozen";
 
 	#[pallet::config]
+	// Avoid as much as possible to tightly couple with other pallets
+	// It seems you are coupling with pallet_sminer just to access ItemLimit constant
+	// You could access that constant through the loosely coupled type MinerControl or
+	// adding a new type for ItemLimit that shares constant with pallet_sminer
```

Comment:
According to your suggestion, I have now made changes here to reduce the coupling between the pallets:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R94